src/components/Pagination/PageSize.tsx

```diff
@@ -49,7 +49,7 @@ class PageSize extends React.PureComponent<IPageSizeProps, any>{
         pageSizeList.map((page: number) => {
           const plain = page !== currentPageSize;
           return <Pager
-                    componentStyle={plain ? { padding: '20px' } : {}}
+                    componentStyle={plain ? { /*padding: '20px'*/ } : {}}
```

Comment:
This method no longer does anything, so should likely be removed

Reply:
As previous message i removed the commented code